### ISSUE TITLE
DateField Ok/Cancel buttons do nothing

### ISSUE THREAD

**Issue by cpietrzykowski** [zippyui/react-date-picker#119]
No error, no console output.

Probably unrelated, but on page load:

    Warning: Unknown props `onClearClick`, `selectDate` on <div> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop

I understand that work is underway to update the component due to the unknown props change in React. I'm only including it to be complete about this issue.

Today/Clear buttons seem to function as expected.

I created a brand new project to test just `react-date-picker`. Here's the package.json that should be the minimal requirements (with the exception of `webpack-dev-server`):

```

{
  "name": "react-date-picker-sample",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "webpack": "webpack-dev-server",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "UNLICENSED",
  "private": true,
  "dependencies": {
    "moment": "^2.14.1",
    "react": "^15.2.1",
    "react-date-picker": "^5.3.20",
    "react-dom": "^15.2.1"
  },
  "devDependencies": {
    "babel-core": "^6.10.4",
    "babel-loader": "^6.2.4",
    "babel-preset-es2015": "^6.9.0",
    "babel-preset-react": "^6.11.1",
    "css-loader": "^0.23.1",
    "style-loader": "^0.13.1",
    "webpack": "^1.13.1",
    "webpack-dev-server": "^1.14.1"
  }
}

```

index.jsx:

```

import React from 'react';
import ReactDOM from 'react-dom';
import 'react-date-picker/index.css';
import { DateField } from 'react-date-picker';

ReactDOM.render(
  <div>
    <DateField
      forceValidDate
      dateFormat="YYYY-MM-DD"
      onChange={(dateFormat, {dateMoment}) => {
        console.log(dateMoment);
      }}
    />
  </div>,
  document.getElementById('app')
);

```

**Comment by radubrehar** [comment]
Yes, again related to #117 and React 15.2.0. Fixed in `5.3.21`, just published to npm.

Thanks for reporting!

**Comment by cpietrzykowski** [comment]
Thanks for the update. All is chill again (usable) in 5.3.21.
